### DISCUSSION TITLE
```forceUtcTimezone: true``` generates ```timestamptz``` type in migrations

### DISCUSSION THREAD

**Discussion by artsiommiksiuk** [mikro-orm/mikro-orm#5354] [General]
### Describe the bug

Setting ```forceUtcTimezone: true``` in mikroOrm config still generates timezone timestamps types in postgresql migrations.

As per [documentation](https://mikro-orm.io/docs/configuration#forcing-utc-timezone), this shouldn't be the case.

### Reproduction

Set forceUtcTimezone: true in mikroOrm config with postgresql driver.

```
@Entity({ tableName: "organization" })
export class OrganizationEntity {
    @PrimaryKey({ type: "uuid" })
    id!: string;

    @Property({ type: "datetime", onCreate: () => new Date() })
    createdAt!: Date;

    @Property({ type: "datetime", onCreate: () => new Date(), onUpdate: () => new Date() });
    updatedAt!: Date;

    @Property()
    name!: string;
}
```

Generate migration for this entity.
In my case it's an altering migration and it gives:

```
this.addSql('alter table "organization" add column "created_at" timestamptz not null, add column "update_at" timestamptz not null;');
```

I'd expect ```timestamp``` type used instead of ```timestamptz```.

### What driver are you using?

None

### MikroORM version

6.1.10

### Node.js version

20.11.1

### Operating system

Mac OS

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/mikro-orm/mikro-orm/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://mikro-orm.io/docs).
- [X] Check that there isn't [already an issue](https://github.com/mikro-orm/mikro-orm/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/mikro-orm/mikro-orm/discussions) or join our [Slack](https://join.slack.com/t/mikroorm/shared_invite/enQtNTM1ODYzMzM4MDk3LWM4ZDExMjU5ZDhmNjA2MmM3MWMwZmExNjhhNDdiYTMwNWM0MGY5ZTE3ZjkyZTMzOWExNDgyYmMzNDE1NDI5NjA).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by B4nan** [comment]
`forceUtcTimezone` has no effect on schema, this flag affects only runtime (hydration and escaping of dates in queries). I don't really like the idea of changing behavior of metadata discovery based on this flag (not to mention it would be breaking). Also, note that it's valid to have some dates in `timestamp` and others in `timestamptz`.

Lastly, the `forceUtcTimezone` actually has an effect on `timestamptz` columns too - it affects all `Date` properties regardless of their "backend".

> As per [documentation](https://mikro-orm.io/docs/configuration#forcing-utc-timezone), this shouldn't be the case.

Sounds like you are reading way in between the lines, that's not what the docs are saying. They say the flag affects `timestamp` columns, not that it forces them to be default `Date` property mapping.

**Reply by artsiommiksiuk** [reply]
Alright, my bad then. How can I make my dates to be timestamp?

**Reply by B4nan** [reply]
You can use `columnType`, that option is never post processed by the ORM - you get what you put inside it. 

```ts
@Property({ columnType: "timestamp", onCreate: () => new Date() })
createdAt!: Date;
```

**Reply by artsiommiksiuk** [reply]
Thank you!)
